frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -45,6 +45,10 @@
 
     static final Logger logger = LoggerFactory.getLogger(WorkerThread.class);
 
+    private static final int[] BACK_OFF = {
```

Comment:
Can you rename this to Fibonacci back off or add a comment indicating the back off strategy is fibonacci ?